### PR TITLE
fix: dropdown container focus styles

### PR DESCRIPTION
**Pull request by mshaaban0**
# Purpose of PR

Refine dropdown container focus style 

**Before**
Screenshot 2021-09-14 at 15 03 47

**After**
Screenshot 2021-09-14 at 15 04 58

## PR Checklist

- [x] I have read the relevant `readme.md` file(s)
- [x] All commits follow our [Git commit message convention](https://github.com/contentful/forma-36/tree/master/packages/forma-36-react-components#commits)
- [x] Tests are added/updated/not required
- [x] Tests are passing
- [x] Storybook stories are added/updated/not required
- [x] Usage notes are added/updated/not required
- [x] Has been tested based on [Contentful's browser support](https://www.contentful.com/faq/about-contentful/#which-browsers-does-contentful-support)
- [x] Doesn't contain any sensitive information

### CODE REVIEW

**Review by suevalov — APPROVED** [review]

**Comment by denkristoffer** [comment]
It doesn't seem like we have design for this state in v4 actually, did you align with @domarku or @fabe ?

**Review by gui-santos — APPROVED** [review]
Cool!

as @denkristoffer suggested
let's wait for @domarku or @fabe to give their 👍

File: packages/forma-36-react-components/src/components/Dropdown/DropdownContainer/DropdownContainer.css [hunk 1]
```diff
@@ -22,6 +22,11 @@
   margin: 0;
 
   z-index: var(--z-index-dropdown);
+
+  &:focus {
+    outline: none;
+    box-shadow: var(--glow-muted), var(--box-shadow-default);
```
Comment (domarku) [inline packages/forma-36-react-components/src/components/Dropdown/DropdownContainer/DropdownContainer.css:28]: @mshaaban0 let's use `glow-primary` here. We will change the Secondary and Transparent buttons' focus glow to use `glow-primary`, too.

Comment (domarku) [reply packages/forma-36-react-components/src/components/Dropdown/DropdownContainer/DropdownContainer.css:28]: Thanks for the fix!

**Review by domarku — CHANGES_REQUESTED** [review]

File: packages/forma-36-react-components/src/components/Dropdown/DropdownContainer/DropdownContainer.css (hunk 1 above)
Comment (mshaaban0) [reply packages/forma-36-react-components/src/components/Dropdown/DropdownContainer/DropdownContainer.css]: @domarku Updated

**Comment by domarku** [comment]
👍

**Review by domarku — APPROVED** [review]

**Pull request merged**